src/main/java/uk/gov/hmcts/reform/dg/docassembly/service/DocmosisApiClient.java

```diff
@@ -55,6 +59,18 @@ public Response render(CreateTemplateRenditionDto createTemplateRenditionDto) th
                 .method("POST", requestBody)
                 .build();
 
-        return httpClient.newCall(request).execute();
+        Response response = null;
+
+        StopWatch stopwatch = new StopWatch();
```

Comment:
why not just have two log statements before and after, that will show the time, and you can check the difference
anyway, I guess this is just for debug purposes and will be removed later?